src/ansible_navigator/ui_framework/colorize.py

```diff
@@ -32,32 +39,33 @@
 
 
 class ColorSchema:
-    """Simple holder for the schema (theme)"""
+    """A storage mechanism for the schema (theme)."""
 
     # pylint: disable=too-few-public-methods
 
-    def __init__(self, schema):
-        """start
+    def __init__(self, schema: dict):
```

Comment:
This is not the correct type for dictionaries. It should be something like `Dict[str, str]` or whatever makes sense for our specific case. Do not use `dict` class as a type, using `Dict`.

Reply:
in this case the schema is a theme loaded from a file, so it would be really tricky to go deeper than one layer....  But based on the only theme we have and it's structure, I think this will work,

If and when we ever allow a user specified color theme, the error checking will need to improved...... alot.  Maybe even with a json schema